books/scss/detail.scss

```diff
@@ -6,9 +6,26 @@
   padding: 1.5rem;
   width: 56rem;
 
-  .book-box-cover {
-    height: 368px;
-    width: 261px;
+  .book-box-images {
+    position: relative;
+
+    .book-box-cover {
+      height: 368px;
+      width: 261px;
+    } 
+
+    .book-box-badge {
+      height: 7.4rem;
+      position: absolute;
+      right: -1.4rem;
+      top: -1rem;
```

Comment:
Always try to avoid negative values to absolute positioning. Change it so you have positive please

Reply:
Have been improved